package.json

```diff
@@ -77,7 +77,10 @@
     "express": "^4.17.1",
     "express-winston": "^4.1.0",
     "fs-extra": "^10.0.0",
+    "http-status-codes": "2.2.0",
+    "immutable": "^4.2.4",
     "js-yaml": "^4.1.0",
+    "kujira.js": "https://github.com/Team-Kujira/kujira.js#911a8acbfa28909cc0ad31a5007ee23fc8fd7c5a",
```

Comment:
is this a specific commit being referenced or the main library? any reason why this lib isn't on npm?

Reply:
Hey @fengtality, thanks for the feedback :)

This one was my mistake it seems, from the official documentation they say to use Github, then I though they were not publishing there, but double checking it here now I've seen they are publishing there too.

I've changed here and ran exhaustive tests and everything seems to be working with the NPM library, also the unit tests were moved to the test-bronze folder, as requested, and they are all passing as well.